README.md

```diff
@@ -85,23 +80,22 @@ You can now run the executable in your app directory:
 bundle exec broadside --help
 ```
 
-`bundler` can also install binstubs for you - small scripts in the `/bin` directory of your application that will mean you can type `bin/broadside do_something` instead of `bundle exec broadside do_something`.  If you want to save the typing, run:
-
+You may also generate binstubs using
```

Comment:
it's only a google away, also i don't think i've seen any project need to explain what it is. similarly, we don't go explaining how bundle or gem install works.